docs/exit_validation.md

```diff
@@ -14,52 +14,52 @@ This document focuses on the Watcher.
-This is OK, as long as the delay doesn't exceed a predetermined `sla_margin`.
+This is acceptable as long as the delay doesn't exceed a predetermined `sla_margin`.
 
 ## Standard Exits
 
 ### The purpose of having exit validation as a separate service is to:
-1. proactively prevent user from losing money
-    - prohibit user from spending on a dangerous chain
-    - try to preempt underfunded chain situation, which would jeopardize user's funds
-2. letting know user about byzantine child chain as reasonably fast as possible
-4. (optional) protect against loss in case of invalid exit finalization
-5. prevent spurious prompts to mass exit, that might result from root chain reorgs, race conditions etc.
+1. proactively prevent the user from losing money by:
+    - prohibiting the user from spending on a dangerous chain
+    - trying to preempt underfunded chain situation, which would jeopardize user's funds
+2. let the user know about byzantine child chain as soon as is reasonably possible
+4. (optional) protect against loss in case of an invalid exit finalization
+5. prevent spurious prompts to mass exit that might result from root chain reorgs, race conditions etc.
 
 ### Actions that the Watcher should prompt:
 1. If an exit is known to be invalid it should be challenged (almost) immediately
 2. If an exit is invalidated with a transaction submitted within an acceptable `sla_margin` period it must be challenged
 3. If an exit is invalidated with a transaction submitted after an acceptable `sla_margin` period it must be challenged **AND** watcher must prompt an exit **AND** watcher must not allow spending and depositing
-    - because our child chain implementation will never get close to this happening, so it is a symptom of a (subtle) hack of the child chain server.
+    - because our child chain implementation will never get close to this happening, so it is a symptom of a (subtle) hack attempt on the child chain server.
     By "subtle" we mean that it is a hack that counts on getting away with exit invalidation being submitted too late, rather than on just dropping a huge invalid UTXO and attempting to exit from that
-3. If an exit is invalid and unchallenged within a short period from it's finalization it must be challenged **AND** watcher must not allow spending and depositing
-    - because otherwise, we are "tightly" following the minimum safety requirements
-    - because otherwise, unchallenged exits may break the chain without honest holders having time to escape.
-    This is the case when an exit, a spend invalidating it and exit of that coincide and are unchallenged.
-4. (optional) If a finalization of an invalid exit occurs, watcher must prompt an exit without guarantees of recovery of funds
+3. If an exit is invalid and remains unchallenged within a short period of time from its finalization, it must be challenged **AND** watcher must not allow spending and depositing, because:
```

Comment:
Question: How can the Watcher prevent deposits? I didn't think the Watcher was involved with deposits at all.

Reply:
it will be. All ethereum-txs making the deposits should be routed through the Watcher which will not allow that if it is unsafe, see OMG-119